src/main/java/ca/corefacility/bioinformatics/irida/model/assembly/UploadedAssembly.java

```diff
@@ -8,18 +8,17 @@
 import org.hibernate.envers.Audited;
 import org.springframework.data.jpa.domain.support.AuditingEntityListener;
 
-import ca.corefacility.bioinformatics.irida.model.VersionedFileFields;
 import ca.corefacility.bioinformatics.irida.model.remote.RemoteStatus;
 
 /**
- * A {@link GenomeAssembly} implementation that was uploaded by a user or service
+ * A {@link GenomeAssembly} implementation that was uploaded by a user or
+ * service
  */
 @Entity
 @Table(name = "uploaded_assembly")
 @EntityListeners(AuditingEntityListener.class)
 @Audited
-public class UploadedAssembly extends GenomeAssembly implements VersionedFileFields<Long> {
-
+public class UploadedAssembly extends GenomeAssembly {
```

Comment:
Removed redundant implementation as `GenomeAssembly` already implements `VersionedFileFields` so it was not required.